model-optimizer/extensions/load/tf/loader.py

```diff
@@ -16,6 +16,8 @@
 
 try:
     import tensorflow.compat.v1 as tf_v1
+    # disable eager execution of TensorFlow 2 environment immediately
+    tf_v1.disable_eager_execution()
```

Comment:
Can you find the first version of TF when the method `disable_eager_execution()` was introduced?

Reply:
I presume it has been introduced at the same moment when tensorflow.compat.v1 introduced. I will double-check

Reply:
@lazarevevgeny, it happened from 1.14: https://github.com/tensorflow/tensorflow/blob/v1.14.0/tensorflow/python/framework/ops.py#L5882

In the previous version I haven't found.